src/igz_controls/components/validating-input-field/validating-input-field.component.js

```diff
@@ -204,7 +212,7 @@
             });
 
             // if provided `bordersMode` attribute is not one of the available values, set it to a default
-            if (!lodash.has(BORDER_MODES, ctrl.bordersMode)) {
+            if (!lodash.values(BORDER_MODES).includes(ctrl.bordersMode)) {
```

Comment:
You're right this was a bug.
But luckily `_.includes` accepts objects as first argument, not only arrays, so we can make this nicer:
```suggestion
            if (!lodash.includes(BORDER_MODES, ctrl.bordersMode)) {
```

Reply:
Please also add such `if` that would assign `defaultFieldType` to `ctrl.fieldType` in case `ctrl.fieldType` is not one of the values in `FIELD_TYPES`.

Reply:
Actually this might not work properly if we assign to `ctrl.fieldType`. Maybe we have to define another property to hold it, since `ctrl.fieldType` comes from `@` binding which might override it back to what the parent component assigned to it.

Reply:
I've just fixed the same issue in `igzAutoComplete` component. You can see here https://github.com/iguazio/dashboard/pull/3704